### PR TITLE
Initial state

### PR DESCRIPTION
**Pull request by LianaHus**

### CODE REVIEW

File: qml/DefaultCompleteButton.qml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+		}
+		label: RowLayout {
+			Image {
+			   source: imageSource
+			   width: 48
+			   height: 48
+			   id: openProjectImg
+			   anchors.top: parent.top
+			}
+			Label
```
Comment (yann300) [inline qml/DefaultCompleteButton.qml]: better use DefaultLabel than Label.
DefaultLabel is mapped to the system font size.

File: qml/MainContent.qml [hunk 2]
```diff
@@ -101,11 +101,12 @@ Rectangle {
 		property alias codeWebOrientation: codeWebSplitter.orientation
 		property alias webWidth: webPreview.width
 		property alias webHeight: webPreview.height
-		property alias showProjectView: projectList.visible
+		//property alias showProjectView: projectList.visible
```
Comment (yann300) [inline qml/MainContent.qml]: can we remove this or will be reused later?

File: qml/Application.qml [hunk 3]
```diff
@@ -333,7 +338,7 @@ ApplicationWindow {
 		id: createProjectAction
 		text: qsTr("&New Project")
 		shortcut: "Ctrl+N"
-		enabled: true;
+		enabled: !projectModel.isEmpty
```
Comment (yann300) [inline qml/Application.qml]: this is the opposite no?

Comment (LianaHus) [inline qml/Application.qml]: this is wrong. I'll revert it

**Comment by yann300** [comment]
rebase needed

**Pull request merged**